packages/io/io-utils/Blob.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Blob.js
+ *
+ * Node and Browserify Compatible
+ *
+ * Copyright (c) 2015 by Z3 Dev (@zdev/www.z3dev.jp)
+ * License: MIT License
+ *
+ * This implementation uses the Buffer class for all storage.
+ * See https://nodejs.org/api/buffer.html
+ *
+ * URL.createObjectURL(blob)
+ *
+ * History:
+ * 2020/10/07: converted to class
+ * 2015/07/02: contributed to OpenJSCAD.org CLI openjscad
+ */
+
+/**
+ * The Blob object represents a blob, which is a file-like object of immutable, raw data; they can be read as text or binary data.
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Blob
+ */
+class Blob {
+  /**
+   * Returns a newly created Blob object which contains a concatenation of all of the data in the given contents.
+   * @param {Array} contents - an array of ArrayBuffer, or String objects that will be put inside the Blob.
+   */
+  constructor (contents, options) {
+    // make the optional options non-optional
+    options = options || {}
+    // number of bytes
```

Comment:
couple of readability comments here...
- Not clear if the comment goes with the line above or below.
- the comments themselves are a smell. Instead of 
```
// media type
this.type = ''
// readability state (CLOSED: true, OPENED: false)
this.isClosed = false
```
why not 
```
this.mediaType = ''
this.isUnreadable = false
```

Reply:
I think those came from the JavaScript specifications. Will double check...

Reply:
Yup. Those attribute names are from the specification.

https://w3c.github.io/FileAPI/#constructorBlob

I tweaked the doc strings, which hopefully make those values slightly clearer.